frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/bs.ts

```diff
@@ -0,0 +1,71 @@
+export enum Providers {
+  AWS = 'AWS S3',
+  AZURE = 'Azure Blob',
+  S3 = 'S3 Compatible',
+  PVC = 'PVC',
+}
+
+const ACC_KEY = 'my_dummy_test_key';
+const SEC_KEY = 'my_dummy_sec_key';
+const TARGET = 'my_dummy_target';
+const ENDPOINT = 'http://test-endpoint.com';
+
+export const testName = `test-${Math.random()
+  .toString(36)
+  .replace(/[^a-z]+/g, '')
+  .substr(0, 5)}-bs`;
+const inputCustomSecrets = () => {
+  cy.byTestID('switch-to-creds').click();
+  cy.byTestID('backingstore-access-key').type(ACC_KEY);
+  cy.byTestID('backingstore-secret-key').type(SEC_KEY);
+  cy.byTestID('backingstore-target-bucket').type(TARGET);
+};
+const setupAWS = () => {
+  cy.byTestDropDownMenu('AWS S3').click();
+  cy.byTestID('backingstore-aws-region-dropdown').click();
+  cy.byTestDropDownMenu('us-east-1').click();
+  inputCustomSecrets();
+};
+
+const setupAzureBlob = () => {
+  cy.byTestDropDownMenu('Azure Blob').click();
+  inputCustomSecrets();
+};
+
+const setupS3Endpoint = () => {
+  cy.byTestDropDownMenu('S3 Compatible').click();
+  cy.byTestID('backingstore-s3-endpoint').type(ENDPOINT);
+  inputCustomSecrets();
+};
+
+const setupPVC = () => {
+  cy.byTestDropDownMenu('PVC').click();
+};
+
+const setupProvider = (provider: string) => {
+  cy.byTestID('backingstore-provider').click();
+  switch (provider) {
+    case Providers.AWS:
+      setupAWS();
+      break;
+    case Providers.AZURE:
+      setupAzureBlob();
+      break;
+    case Providers.S3:
+      setupS3Endpoint();
+      break;
+    case Providers.PVC:
+      setupPVC();
+      break;
+    default:
+      break;
+  }
+};
+
+export const bs = {
+  createStore: (provider: string) => {
```

Comment:
### 
```suggestion
  createStore: (provider: Providers) => {
```
maybe?

Reply:
since we're changing for the setup, we have to change this